app/javascript/chat/ChatChannelSettings/ChannelDescriptionSection.jsx

```diff
@@ -4,25 +4,21 @@ import PropTypes from 'prop-types';
 const ChannelDescriptionSection = ({
   channelName,
   channelDescription,
-  currentMembershipRole
+  currentMembershipRole,
 }) => {
   return (
     <div className="p-4 grid gap-2 crayons-card mb-4 channel_details">
-      <h1 className="mb-1 channel_title">{channelName}</h1>
+      <h2 className="mb-1 channel_title">{channelName}</h2>
```

Comment:
I'm not sure if replacing H1 with H2 is good in terms of accessibility - I'm not saying we use H* tags correctly but this feels more like workaround and not real fix. 

I'd much more recommend updating the style of H1 tag to make it smaller if that's the idea.

Reply:
I will try to change it.

Reply:
Also, I think we need to limit the length of `channelName`

Reply:
I will set the default tag as h2, and if the screen width is more than 1300px, then set the h2 tag bigger which same as h1 tag.

Reply:
@ludwiczakpawel ping to circle back here?